android/java/org/chromium/chrome/browser/brave_news/BraveNewsBottomSheetDialogFragment.java

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
this file needs `git cl format android/java/org/chromium/chrome/browser/brave_news/BraveNewsBottomSheetDialogFragment.java`

Reply:
done